src/NCPopupMenu.cc

```diff
@@ -75,15 +90,29 @@ NCursesEvent NCPopupMenu::wHandleInput( wint_t ch )
 
 		if ( tableItem )
 		{
-		    YMenuItem * item = itemsMap[ tableItem ];
+		    auto current = find_item(tableItem);
+
+		    if (current != _items.end())
+		    {
+			YMenuItem * item = (*current)->menu_item;
 
-		    if ( item && item->hasChildren() )
-			ret = NCursesEvent::button;
+			if ( item && item->hasChildren() && item->isEnabled() )
```

Comment:
Can a disabled item ever cause this event?
Does this mean that the user has to skip over disabled items manually?
I hope they are automatically skipped.

Reply:
When all items of a menu are disabled, the current item points to a disabled item. I will try to clarify this in the code.